platform.js

```diff
@@ -190,10 +199,15 @@ module.exports = {
   },
   launchExplorer() {
     if (module.exports.isWindows() && module.exports.getWindowsVersion() !== "7") {
+      const cwd = process.env.SystemRoot ?
+        process.env.SystemRoot :
+        path.join(module.exports.getCwdRoot(), "Windows");
```

Comment:
SystemRoot should always be [drive]:\Windows but just in case it isn't, we'll take the drive letter from whatever the current working dir is.